### PR TITLE
{bio}[foss/2018b] bnpy v0.1.6 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3ed8ad5d521a062285f2fa98bf56d6ae for a full test report.

File: easybuild/easyconfigs/p/Pillow/Pillow-5.3.0-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '5.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
```
Comment (Micket) [inline easybuild/easyconfigs/p/Pillow/Pillow-5.3.0-foss-2018b-Python-2.7.15.eb]: https

File: easybuild/easyconfigs/p/Pillow/Pillow-5.3.0-foss-2018b-Python-2.7.15.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+]
+
+download_dep_fail = True
+use_pip = True
+
+options = {'modulename': 'PIL'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```
Comment (Micket) [inline easybuild/easyconfigs/p/Pillow/Pillow-5.3.0-foss-2018b-Python-2.7.15.eb]: This is the default in PythonPackage, you can skip sanity_check_paths

Comment (lexming) [reply easybuild/easyconfigs/p/Pillow/Pillow-5.3.0-foss-2018b-Python-2.7.15.eb]: True, fixed :+1:

**Comment by Micket** [comment]
Going in, thanks @lexming!

**Pull request merged**